### PR TITLE
Add colours and links at tables

### LINKED ISSUE #148: Colours 🌈 🌟
I'm a big fan of colours in the terminal, I find it makes outputs much easier to read.

I wonder if some of the tower cli outputs could be improved with a dash of colour. For example, highlighting statuses and varying column colours a little when listing runs would make it much easier to scan:

image

We try to do this with the @nf-core helper tools. We're in Python and I use the `rich` library, which also allows having URLs linked to from text. For example, cmd+clicking the pipeline name in this output links to the webpage for that pipeline:

image

### PR DESCRIPTION
**Pull request by jordeu**
### Description
Use colours at some tables. Resolves #148.

### General
- [x] use bold at table headers.

### Pipelines
- [x] list: link name to edit
- [x] view: link 

### Runs
- [x] list: link workflow id
- [x] list: colour status
- [x] view: link workflow id
- [x] view: colour status

### Actions
- [x] list: colour status
- [x] list: link ID to edit page
- [x] view: colour status
- [x] view: link ID

### Compute-envs
- [x] list: colour status
- [x] list: link ID to view page
- [x] view: colour status
- [x] view: link ID

### Credentials
- [x] list: link ID to edit page

### Participants
- [x] list: colour participant type

### Members
- [x] list: colour role

### Teams
- [x] list: link ID to teams members list

### Workspaces
- [x] list: link ID to workspace launchpad
- [x] list: link organization ID to organization view

### Organizations
- [x] list: link ID to organization view
- [x] view: link ID

### CODE REVIEW

File: src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java [hunk 1]
```diff
@@ -38,9 +48,14 @@ public void toString(PrintWriter out) {
             return;
         }
 
-        TableList table = new TableList(out, 5, "ID", "Status", "Platform", "Name", "Last activity").sortBy(0);
+        TableList table = new TableList(out, 5, "  ID", "Status", "Platform", "Name", "Last activity").sortBy(0);
```
Comment (grananda) [inline src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java:51]: Why the extra space in the ID header?

Comment (jordeu) [reply src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java:51]: To align the start of the header with the start of the cell. Here the cells have an extra `*` or ` ` at the beginning to mark the primary compute environment.

**Review by grananda — APPROVED** [review]
This was endless :)

File: src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java (hunk 1 above)
Comment (grananda) [reply src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java:51]: Good enough, thanks

**Pull request merged**